### PR TITLE
london fork

### PR DESCRIPTION
**Pull request by temaniarpit27**

### CODE REVIEW

**Review by ssandeep — APPROVED** [review]

File: params/config.go [hunk 1]
```diff
@@ -253,7 +253,7 @@ var (
 			ValidatorContract:     "0x0000000000000000000000000000000000001000",
 			StateReceiverContract: "0x0000000000000000000000000000000000001001",
 			BurntContract: map[string]string{
-				"0": "0x0000000000000000000000000000000000000000f",
+				"0": "0x00000000000000000000000000000000000000000",
```
Comment (ssandeep) [inline params/config.go:256]: Why this change?

Comment (temaniarpit27) [reply params/config.go:256]: this is some random address which i had put earlier
its for only testcases

**Review by ssandeep — APPROVED** [review]
